docs/farming-&-staking/staking/operators/register-operator.mdx

```diff
@@ -227,70 +226,43 @@ Prefer a video? Expand for our installation video using PolkadotJS interface.
 
 ![NStaking-3](/img/doc-imgs/operators-staking/NStaking-3.png)
 
-4. Select the account you'd like to use form the dropdown menu. You will see both available and locked (staked) token balances for each account. 
+4. Select the account you'd like to use form the dropdown menu. You can click the button in the upper left corner to see available and locked (staked) token balances for the selected account. 
 
 ![NStaking-4](/img/doc-imgs/operators-staking/NStaking-4.png)
-5. Proceed to the `Stake as a pool operator` tab.
 
 ![NStaking-5](/img/doc-imgs/operators-staking/NStaking-5.png)
-6. Select the `domainId` you would like to be registered on. Enter the `Minimum Operator Stake`, `Amount to Stake`, `Nomination Tax` and `Signing key` and then click `Next`.
+
+5. Proceed to the `Register Operator` tab.
 
 ![NStaking-6](/img/doc-imgs/operators-staking/NStaking-6.png)
 
-:::info
-Make sure to use the signing key generated on the previous **[Create operator key](#create-operator-key)** step.
-:::
-7. Approve the request in the pop-up window. 
+6. Select the `domainId` you would like to be registered on.  For Stake Wars, Nova is Domain 0 and Auto-ID should be Domain 1 once it is available.
+
+![NStaking-7](/img/doc-imgs/operators-staking/NStaking-7.png)
 
-![NStaking-8](/img/doc-imgs/operators-staking/NStaking-7.png)
-8. Congratulations, you're now registered as an **operator**! 
+7. An important security change from Stake Wars one is that you are now required to have a Proof of signing key ownership signature.  This is derived from your seed phrase when creating the operator key.  
+You can either select the file that contains your seed in the subfolder of **NODE_DATA_PATH** under `/domains/domainID/keystore` and then click the choose file and point to that file or you can choose the Proof with seed and manually enter the seed phrase.
+Once you click the Generate proof button it will automatically populate the `Signing key` and `Prof of signing key ownership signature` fields.  Enter the `Minimum Operator Stake`, `Amount to Stake`, `Nomination Tax` and then click `Next`.
```

Comment:
```suggestion
Once you click the Generate proof button it will automatically populate the `Signing key` and `Proof of signing key ownership signature` fields.  Enter the `Amount to Stake`, `Nominator Tax`, `Minimum Nominator Stake` and then click `Next`.
```

Reply:
I think we should also add a note here that the minimum nominator stake [couldn't be lower than 1TSSC](https://academy.autonomys.net/subspace-protocol/decoupled-execution/staking#nomination-pools) if that change has made it into the code already.